apps/web/components/eventtype/EventAppsTab.tsx

```diff
@@ -103,7 +105,6 @@ export const EventAppsTab = ({ eventType }: { eventType: EventType }) => {
               },
             }}
             eventType={eventType}
-            {...shouldLockDisableProps("apps")}
```

Comment:
Removed switch from each app as it referred to the general apps locked state now present in the alert